ui/mods/mod_reforged/js_hooks/screens/character/modules/character_screen_left_panel/character_screen_stats_module.js

```diff
@@ -3,6 +3,7 @@ Reforged.Hooks.CharacterScreenStatsModule_setProgressbarValues = CharacterScreen
 CharacterScreenStatsModule.prototype.setProgressbarValues = function (_data)
 {
 	Reforged.Hooks.CharacterScreenStatsModule_setProgressbarValues.call(this, _data);
+	this.setProgressbarValue(this.mLeftStatsRows.Morale.Progressbar, _data, ProgressbarValueIdentifier.rf_Reach, ProgressbarValueIdentifier.rf_ReachMax, ProgressbarValueIdentifier.rf_ReachLabel);
```

Comment:
What is this line for? 
We are not using progressbars to display reach, yet you call this vanilla function, which we don't have hooked in reforged.

Reply:
We replace the Morale bar on character screen with Reach.

Reply:
I see. 
I was fixated on the combat overlay only and couldn't make sense of it